reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/greenfield.json

```diff
@@ -309,7 +306,7 @@
       "DependsOn": ["Nat"],
       "Type": "AWS::EC2::Subnet",
       "Properties": {
-        "AvailabilityZone": {"Fn::Select": ["2", {"Ref": "SubnetAvailabilityZones"}]},
+        "AvailabilityZone": {"Fn::Select": ["2", { "Fn::GetAZs" : "" }]},
```

Comment:
I think this causes issues when the account is an AWS Classic account and using a VPC, pretty sure it will return all AZs in that case, even ones not VPC-enabled.